src/WP_CLI/NonDestructiveCoreUpgrader.php

```diff
@@ -8,7 +8,7 @@
  * @package wp-cli
  */
 class NonDestructiveCoreUpgrader extends CoreUpgrader {
-	function unpack_package($package, $delete_package = false) {
+	private function unpack_package( $package, $delete_package = false ) {
```

Comment:
Just FYI: this is a BC-break as the method was previously `public` (by default). I leave it up to your judgement if changing this is safe.